ansible/files/runtimes.json

```
@@ -208,6 +208,22 @@
             }
         ]
     },
+    "golang": [
+        {
+            "kind": "golang:1.11",
```

this needs to use `go`
```suggestion
            "kind": "go:1.11",
```